tests/MakersOfDenmark.WebAPI.Tests/SearchForMakerSpaceHandlerTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using MakersOfDenmark.Application.Queries.V1;
+using MakersOfDenmark.Domain.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace MakersOfDenmark.WebAPI.Tests
+{
+    public class SearchForMakerSpaceHandlerTest: IClassFixture<RequestHandlerFixture>
+    {
+        private RequestHandlerFixture _requestHandlerFixture;
+        public SearchForMakerSpaceHandlerTest(RequestHandlerFixture requestHandlerFixture)
+        {
+            _requestHandlerFixture = requestHandlerFixture;
+        }
+
+        [Fact]
+        public async Task SearchForOneMakerSpaceTest()
+        {
+            //Arrange
+            var makerSpace = _requestHandlerFixture.Fixture.Build<MakerSpace>().With(x => x.Address, new Address("Test Street", "Test City", "Test Country", "Test Postcode")).Without(x => x.ContactInfo).Without(x => x.VATNumber).Without(x => x.Organization).Without(x => x.Tools).Create();
+
```

Comment:
Du bør tilføje et makerspace mere, så du sikrer dig at koden ikke bare returnerer listen med kun et makerspace i.

Reply:
Du mener for at sikre sig, at den ikke tager MakerSpaces med, som den ikke skal?

Reply:
Når du kun tilføjer eet makerspace til testen, så kunne den jo bare lave en getall og testen består stadig, men du har faktisk ikke tjekket om din søgefunktion fungerer :)

Reply:
God pointe, jeg tilføjer et makerspace eller to mere!